src/pages/CoinPage.jsx

```diff
@@ -406,8 +406,8 @@ const CoinPage = () => {
                   {singleCoin.links &&
                     singleCoin.links.homepage
                       .filter((e) => e !== "")
-                      .map((item, index) => (
-                        <div key={index}>
+                      .map((item) => (
+                        <div key={Math.random()}>
```

Comment:
Every time a component's key changes React will [create a new component instance rather than update the current one](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key), so for performance's sake using Math.random() will be sub-optimal to say the least. Please watch the video in the above comment.